### DISCUSSION TITLE
Object Drawing Mode

### DISCUSSION THREAD

**Discussion by CosmicFail** [keiwando/toonsquid#562] [Ideas]
If you don't know what object drawing mode is I'll briefly explain it. Object drawing mode (Atleast for adobe animate) is something you can toggle on and off that makes it so when you make multiple brush strokes you can select them separately even if they're on the same layer as eachother, or touching eachother. I think this would be useful in a variety of ways such as if you make a brush stroke and you want to move it just a tad, you don't have to lasso it which may look a bit weird. And it also can speed up the process a bit making it so you don't have to constantly which between animation layers. Anyway, Just thought I'd put this here as it's a feature I'd honestly really like to see implemented. Anyway, That's all.

**Comment by kavlor** [comment]
Sounds like something it kinda already has. You have your animation layers which are at the bottom but to the right you access the drawing layers which you can add further drawing layers. If it created a new drawing layer each time you lifted and put down the stylus which it doesn't do (you have to manually click the + for a new drawing layer) you can then click the different drawing layers without leaving the drawing area. it seems like it would be effectively the same thing as your idea.
